sumpy/kernel.py

```diff
@@ -856,6 +857,91 @@ def get_args(self):
                     )
                 ]
 
+
+class ElasticityHelperKernel(ExpressionKernel):
+    """A helper kernel for the explicit solution to half-space Elasticity problem.
+    This kernel is the line of compression or dilatation of constant strength
+    along the axis "icomp" from zero to negative infinity. See [1] for details.
```

Comment:
I was going by Gimbutas et. al. which doesn't name this kernel, but Mindlin does name this kernel.
Shall I keep the name `ElasticityHelperKernel` or name it something like `LineOfCompressionKernel`?

Reply:
I like `LineOfCompression` a lot better than `ElasticityHelper`. :) Include the references for good measure.